modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/service/SubscriptionDLAppHelperLocalServiceWrapper.java

```diff
@@ -301,6 +323,9 @@ else if (commandUpdate &&
 	@Reference
 	private Language _language;
 
+	private LiferayJSONDeserializationWhitelist
+		_liferayJSONDeserializationWhitelist;
```

Comment:
This is unititialized. You need to add the @Reference annotation so that the value can be injected.

Reply:
My bad. I updated the code. Thanks!